Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContainer.js

```diff
@@ -1,2 +1,22 @@
 import React from 'react';
+// import ReactDOM from "react-dom";
+
 import './Header.css';
+
+import HeaderContent from "./components/HeaderComponents/HeaderContent";
+import HeaderTitle from "./components/HeaderComponenets/HeaderTitle";
+import ImageThumbnail from "./components/HeaderComponents/ImageThumbnail";
```

Comment:
Not sure if you were getting errors here, but I was getting a compiler error. Since these components are stored in the same directory as HeaderContainer.js, you can import them simply like this:

```suggestion
import HeaderContent from "./HeaderContent";
import HeaderTitle from "./HeaderTitle";
import ImageThumbnail from "./ImageThumbnail";
```